### ISSUE TITLE
sc3 unselected by default

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#272]
**Reported by efocht on 25 Oct 2006 09:25 UTC**
The sc3 package is actually required by sync_files, and is a core package.
In a test install of trunk on rhel4-i386 Selector displays it as unselected.
This is strange and should be fixed.

**Comment by dikim33** [comment]
**Comment by efocht on 25 Oct 2006 09:36 UTC**
Research shows that this bug was introduced in the changeset:

```
------------------------------------------------------------------------
r5310 | valleegr | 2006-08-21 07:07:32 +0200 (Mo, 21 Aug 2006) | 5 lines
Genderte Pfade:
   M /trunk/scripts/populate_default_package_set

+ mechanism to specify excluded packages from the default package set adding
file in share/package_set. For that, create a file with the pattern
"distro-version-arch.txt" which contains the list of packages that have to be
excluded (one package per line).
-------------------------------------------------------------------------
```

**Comment by dikim33** [comment]
**Comment by efocht on 25 Oct 2006 09:51 UTC**
(In [5435]) Re-added sc3 to installable packages. Don't know why this was removed by
Geoffroy. It needs to be there!
Closes #272
